db/job_context.h

```diff
@@ -142,8 +149,12 @@ struct JobContext {
   // a list of memtables to be free
   autovector<MemTable*> memtables_to_free;
 
+  // context for installing superversion for one column family
   SuperVersionContext superversion_context;
 
+  // contexts for installing superversions for multiple column families
+  std::vector<SuperVersionContext> superversion_contexts;
```

Comment:
who sets superversion_contexts?

Reply:
In [PR 3752](https://github.com/facebook/rocksdb/pull/3752), we have an overloaded `InstallSuperVersionAndScheduleWork` which calls `NewSuperVersion()` for each element of `superversion_contexts`.

Reply:
In the `atomic_flush_=false` case, can we have `superversion_contexts` be a vector of length one to eliminate the redundant variable?

Reply:
Should be doable. Will update code.

Reply:
Done.